scripts/compare_benchmarks.py

```diff
@@ -2,15 +2,23 @@
 
 import json
 import sys
-from beautifultable import BeautifulTable
+from terminaltables import AsciiTable
 from termcolor import colored
+from scipy.stats import ttest_ind
```

Comment:
As discussed, we should somehow add these to our dependencies.

Reply:
True. I added it to the discussion part of the PR description. I think I'd still want to have one single DEPENDENCIES.md, but having maybe multiple tables there. Like one for absolutely required stuff to build and run it and one with optional dependencies for development. And for the second one, maybe one for C++ and one for Python?